packages/jana2/package.py

```diff
@@ -58,14 +58,18 @@ class Jana2(CMakePackage):
 
     variant("root", default=False, description="Use ROOT for janarate.")
     variant("zmq", default=False, description="Use zeroMQ for janacontrol.")
+    variant("podio", default=False, description="Use Podio.", when="@2.0.9:")
 
     depends_on("cmake@3.16:", type="build")
     depends_on("cppzmq", when="+zmq")
     depends_on("root", when="+root")
+    depends_on("podio@0.16.:", when="@2.0.9: +podio")
```

Comment:
Minor typo (I don't even know if it matters, but it's not canonical).
```suggestion
    depends_on("podio@0.16:", when="@2.0.9: +podio")
```